state_machine/states/impl/odlc_impl.py

```diff
@@ -1,16 +1,99 @@
 """Implements the behavior of the ODLC state."""
 import asyncio
+from ctypes import c_bool
```

Comment:
Are the c_bool types necessary for cooperation from the camera firmware?

Reply:
To be able to used shared memory like the Value from the multiprocessing library, it has to be typed with ctypes because that is what the library uses, so c_bool is only imported to be used for typing